enhancements/update/change-management-and-maintenance-schedules.md

```diff
@@ -0,0 +1,1220 @@
+Major enterprise users of our software do not update on a predictable, recurring window of time. Updates
+require laborious certification processes and testing. Maintenance schedules will not serve these customers
+well. However, these customers may still benefit significantly from the change management concept --
+unexpected / disruptive worker node drains and reboots have bitten even experienced OpenShift operators
+(e.g. a new MachineConfig being contributed via gitops).
+
+These strategies inform decision-making through metrics and provide facilities for fine-grained control 
+over exactly when material change is rolled out to a cluster. 
+
+The Assisted strategy is also specifically designed to provide a foundation for 
+the forthcoming `oc adm update worker-nodes` verbs. After separating the control-plane and
+worker-node update phases, these verbs are intended to provide cluster lifecycle administrators the 
+ability to easily start, pause, cancel, and even rollback worker-node changes.
+
+Making accommodations for these strategies should be a subset of the overall implementation
+of the MaintenanceSchedule strategy and they will enable a foundation for a range of 
+different persons not served by MaintenanceSchedule.
+
+### Use CRON instead of RRULE
+The CRON specification is typically used to describe when something should start and 
+does not imply when things should end. CRON also cannot, in a standard way,
+express common semantics like "The first Saturday of every month."
+
+### Use a separate CRD instead of updating ClusterVersion, MCP, ...
+In this alternative, we introduce a CRD separate from ClusterVersion, MCP, HostedCluster, and NodePool. For example
+an independent `UpdatePolicy` CRD where administrator preferences can be captured. This approach [was explored](https://github.com/jupierce/oc-update-design-ideas/commit/a6364ee2f2c1ebf84ed6d50bc277f9736bf793bd). 
+Ultimately, it felt less intuitive across profiles. Consider a CRD called `UpdatePolicy` that tries to be a central config.
+
+1. Is it namespaced? If it is, that feels odd for an object that that should control the cluster. If it is not namespaced, the policy feels misplaced for HCP HostedCluster resources which live in a namespace.
+1. Where does someone check the status of the policy (e.g. when the next update is going to be possible on a given MCP?). If it is the UpdatePolicy object, you have multiple independent controllers controlling the status, which is an antipattern. If the UpdatePolicy controls multiple different MCPs differently, how do they independently report their status'? It also introduces the problem of having to look at multiple places (MCP and UpdatePolicy to understand what may be happening.
+1. As you pack policies for control-plane, MCPs, HCP primitives into an expressive UpdatePolicy, the schema options varied from too complex, to error prone, to overly abstract, to overly limiting.

```

Comment:
I feel like I need to read the other version of this EP now... A next week problem